app/src/main/java/fr/free/nrw/commons/nearby/WikiFeedback.java

```diff
@@ -0,0 +1,35 @@
+package fr.free.nrw.commons.nearby;
+
+import android.os.Bundle;
+import android.widget.TextView;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+
+import androidx.appcompat.app.AppCompatActivity;
+import butterknife.BindView;
+import fr.free.nrw.commons.R;
+import fr.free.nrw.commons.mwapi.MediaWikiApi;
+
+public class WikiFeedback extends AppCompatActivity {
+TextView textView;
+private static final String headerString1="Write something about the ";
+    private static final String headerString2=" item. It will be publicly visible.";
```

Comment:
These two strings form what is know as "lego messages", please read #2498 to see why it is problematic and how you can easily fix them. Thanks! :-)